suite-native/atoms/src/Button/Button.tsx

```diff
@@ -1,118 +1,196 @@
 import React from 'react';
-import { TouchableOpacity, TouchableOpacityProps, View } from 'react-native';
+import { Pressable, PressableProps, View } from 'react-native';
 
-import { Color } from '@trezor/theme';
+import { MergeExclusive } from 'type-fest';
+
+import { Color, TypographyStyle } from '@trezor/theme';
 import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { Icon, IconName } from '@trezor/icons';
 
 import { Text } from '../Text';
 
 export type ButtonSize = 'small' | 'medium' | 'large';
-export type ButtonColorScheme = 'primary' | 'gray' | 'red';
+export type ButtonColorSchemeName = 'primary' | 'secondary' | 'tertiary' | 'danger';
```

Comment:
Please use color naming => forest, green, gray, red (or danger)

Reply:
Please no, this naming follows naming from Figma and we should differ as less as possible.